### DISCUSSION TITLE
Dicas de plano de leitura

### DISCUSSION THREAD

**Discussion by samuel-apls** [clube-de-leitura-python/primeiro_grupo_de_2022.py#2] [Ideas]
19 dias úteis para 6 capítulos (139 páginas ou 70 folhas):

a) Essencial: 09 páginas por dia
b) Desafiador: 02 capítulos por semana

**Comment by omyasuda** [comment]
Realmente, bem _desafiador_!

Mas, contando com a ajuda de todos, tenho a expectativa de que conseguiremos uma boa experiencia de leitura!
